bucket/gog-galaxy.json

```diff
@@ -0,0 +1,50 @@
+{
+    "version": "2.0.23.4",
+    "description": "GOG Galaxy 2.0. Gaming Client designed for a convenient purchasing, playing and updating games, as well as an online play between gaming platforms.",
+    "homepage": "https://www.gog.com/galaxy",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://support.gog.com/hc/en-us/articles/212632089-User-Agreement"
+    },
+    "depends": [
+        "innounp",
+        "sudo"
+    ],
+    "url": "https://cdn.gog.com/open/galaxy/client/setup_galaxy_2.0.23.4.exe",
+    "hash": "md5:0f9284b653e1dc377368678a1bc18582",
+    "installer": {
+        "script": [
+            "$file = \"$dir\\$fname\"",
+            "Expand-InnoArchive $file -ExtractDir '{%ALLUSERSPROFILE}' -DestinationPath $env:ALLUSERSPROFILE",
+            "Expand-InnoArchive $file -Removal"
+        ]
+    },
+    "uninstaller": {
+        "script": [
+            "if (Test-Path $env:LOCALAPPDATA\\GOG.com) { Remove-Item -Recurse -Force $env:LOCALAPPDATA\\GOG.com }",
+            "sudo Remove-Item -Recurse -Force $env:ALLUSERSPROFILE\\GOG.com",
```

Comment:
Sorry, admin rights installation is not allowed in this bucket. No need to do this if you don't extract files to `$env:ALLUSERSPROFILE` while installation.